src/QsCompiler/CompilationManager/ProjectManager.cs

```diff
@@ -1111,7 +1111,9 @@ editCollection switch
         /// </remarks>
         public Hover? HoverInformation(TextDocumentPositionParams? param, MarkupKind format = MarkupKind.PlainText) =>
             this.Manager(param?.TextDocument?.Uri)?.FileQuery(
-                param?.TextDocument, (file, c) => file.HoverInformation(c, param?.Position?.ToQSharp(), format), suppressExceptionLogging: true);
+                param?.TextDocument,
```

Comment:
It feels like there a lot of changes without adding tests. Is the code covered in this PR already under test?

Reply:
The language server has a very limited test suite - mostly, it tests that it can load projects and track edits correctly. There is no test infrastructure for hover information, finding references, etc.

Reply:
While there is definitively a testing gap that we know of and are tracking (within the QDK team), it would be good to add a basic test case specifically for scenarios we are adding. See e.g. the test added for the new formatting support here: https://github.com/microsoft/qsharp-compiler/pull/1207/files#diff-b160aef5299bccdebd1263eafd99d9d7811162a4ac2bc45cc007d318afdb5c02
The coverage won't be perfect, but some coverage is better than none.

Reply:
OK, so you mean add tests to the feature/lambda branch before it's merged to main, but tests are not necessary for this PR?

Reply:
Yes, I was suggesting to add a test case for the hover and find all references for lambdas before merging to main, but not necessarily as part of this PR (in the spirit of testing the functionality rather than the implementation, even though some of these are technically part of a public API).